app/src/views/Contribution.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <template v-if="fullContributions && !loading">
+    <!-- this is a header for a the route "/contributions" what show ALL contributions.
+      other pages already have a header, so its not needed there. -->
+    <BaseHeader :name="title" />
+
+    <!-- btw in this component was a "cursor-pointer" for some reason. i removed it as this is
+    not something clickable. just for final PR this component could need that update too-->
+    <SectionHeader :title="`Contributions (${contributionTotal})`" />
+
+    <!-- a single contribution ( right now this would be the "ContributionRow.vue" what we maybe
+    not even wana keep - depends on the architecture you wana do ...
+    i added a v-for loop with some dummy data for {{name}} just to test how this looks
+    you can delete this of cause ...
+    -->
```

Comment:
^ do we need to clean up these comments ?

Reply:
@thelostone-mc we do not! Will remove